cafebabel/templates/users/edit.html

```diff
@@ -1,7 +1,7 @@
 {% from "users/macros.html" import social_field, default_profile_picture with context %}
 {% extends 'base.html' %}
 
-{% block extra_js %}<script src={{ url_for('static', filename='js/edit-profile.js') }}></script>{% endblock %}
+{% block extra_js %}<script src={{ url_for('static', filename='js/image-preview.js') }}></script>{% endblock %}
```

Comment:
This extra_js block is not using the macro you had written.

Reply:
ah oui. my bad